### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by chilampoon**
README & vignette updates

### CODE REVIEW

**Review by SolomonShorser-OICR — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
-## ReactomeContentService4R: an R interface for Reactome Content Service
+## ReactomeContentService4R: an R interface for the Reactome Content Service
 
 <!-- badges: start -->
 [![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
-[![Build Status](https://travis-ci.com/reactome/reactome4r.svg?token=qrkG3qhYuEKpFXjXW1Fp&branch=develop)](https://travis-ci.com/reactome/reactome4r)
+[![Build Status](https://travis-ci.com/reactome/ReactomeContentService4R.svg?branch=master)](https://travis-ci.com/reactome/ReactomeContentService4R)
```
Comment (cookersjs) [inline README.md:5]: Does this mean that we will need to merge this into Master?

**Review by cookersjs — APPROVED** [review]
I only had one question about the specification of master vs develop. Please answer that, but I am going to approve anyways

**Comment by chilampoon** [comment]
> I only had one question about the specification of master vs develop. Please answer that, but I am going to approve anyways

Seems it doesn't matter because I just need to submit a `tar.gz` file. I will do more checks for the submission later

**Comment by chilampoon** [comment]
There were some weird warnings from the vignette image generation in checks for windows and I can't find a way to solve them... so I just saved the plots into a [folder](https://github.com/reactome/ReactomeContentService4R/tree/feature/updates/vignettes/img) and capture them in the vignette

**Review by cookersjs — DISMISSED** [review]

**Comment by chilampoon** [comment]
There is a warning: `"The following files are over 5MB in size: '.git/objects/pack/pack-e5c28359d60f7df0fe149b898663be16f1d8572c.pack'"`
Can this file be removed?

**Comment by SolomonShorser-OICR** [comment]
Where did that warning come from? What triggered it?

**Comment by chilampoon** [comment]
> Where did that warning come from? What triggered it?

It's the same as this [issue](https://github.com/Bioconductor/BiocCheck/issues/39), but that warning won't appear if BiocCheck() a tarball!

**Comment by SolomonShorser-OICR** [comment]
> > Where did that warning come from? What triggered it?
> 
> It's the same as this [issue](https://github.com/Bioconductor/BiocCheck/issues/39), but that warning won't appear if BiocCheck() a tarball!

Ah. I'm not very familiar with `BiocCheck()`. I don't know why it is interested in the contents of .git. Does it use the .git files for something specific?

**Comment by chilampoon** [comment]
> > > Where did that warning come from? What triggered it?
> > 
> > 
> > It's the same as this [issue](https://github.com/Bioconductor/BiocCheck/issues/39), but that warning won't appear if BiocCheck() a tarball!
> 
> Ah. I'm not very familiar with `BiocCheck()`. I don't know why it is interested in the contents of .git. Does it use the .git files for something specific?

One of the bioconductor software package requirements is the size for all individual files no larger than 5 MB...

**Comment by SolomonShorser-OICR** [comment]
> 
> One of the bioconductor software package requirements is the size for all individual files no larger than 5 MB...

Yes, but why is it looking at the contents of .git? It seems strange to me that bioconductor cares about a project's history in git. I guess you could move .git to some other location temporarily, and the move it back, if bioconductor doesn't actually _need_ the contents of .git.

**Comment by chilampoon** [comment]
> > One of the bioconductor software package requirements is the size for all individual files no larger than 5 MB...
> 
> Yes, but why is it looking at the contents of .git? It seems strange to me that bioconductor cares about a project's history in git. I guess you could move .git to some other location temporarily, and the move it back, if bioconductor doesn't actually _need_ the contents of .git.

Not sure why .git is checked as well... Following this guide http://bioconductor.org/developers/how-to/git/remove-large-data/, I found the html files in this branch `feature/vignette-review` make the `.pack` file in `.git` > 5Mb. I deleted the vignette branch then the `.pack` becomes smaller (<5Mb)...

**Review by cookersjs — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@
 language: R
 cache: packages
 
+bioc_check: true
```
[the remark this answers is not in the document]
Comment (chilampoon) [reply .travis.yml:6]: The biocCheck reports issues that fail to meet the requirements: https://bioconductor.org/packages/devel/bioc/vignettes/BiocCheck/inst/doc/BiocCheck.html

**Pull request merged**